code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -518,3 +506,89 @@
 	id = "laughter"
 	results = list("laughter" = 10)
 	required_reagents = list("sugar" = 1, "banana" = 1)
+
+//////////////////////////////////// Solidification ///////////////////////////////////////////
+
+/datum/chemical_reaction/plasmasolidification
+	name = "Solid Plasma"
+	id = "solidplasma"
+	required_reagents = list("pyrosium" = 10, "cryostylane" = 10, "plasma" = 20)
+	mob_react = 1
+
+/datum/chemical_reaction/plasmasolidification/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i in 1 to created_volume)
+		new /obj/item/stack/sheet/mineral/plasma(location)
+	return
+
+/datum/chemical_reaction/metalsolidification
```

Comment:
I remade the last pull request because it was merged before I could sync another change